src/StaticCategory/StaticCategory.js

```diff
@@ -32,17 +34,21 @@ const StaticCategory = ({ catTitle, items, openModal, verifyDeletion }) => {
     return category.icon
   }
 
+  const toggleListItems = () => {
+    setDisplayItems(!displayItems)
+  }
+
   return (
     <section className='category-container'>
         <h1 className='cat-title'>
           {generateCategoryIcon(catTitle)}
           <div className='edit-category-container'>
             {catTitle}
-            <button className='edit-category-btn'><BsPlusCircle size={20} /></button>
+            <button className='edit-category-btn' onClick={toggleListItems}>{displayItems === true ? <FiMinusCircle size={20} /> : <FiPlusCircle size={20} />}</button>
```

Comment:
Can we make this user experience more obvious with arrows or something?

Reply:
Done!